bigquery/storage/managedwriter/integration_test.go

```diff
@@ -42,18 +42,13 @@ var (
 	defaultTestTimeout = 30 * time.Second
 )
 
-var testSimpleSchema = bigquery.Schema{
-	{Name: "name", Type: bigquery.StringFieldType, Required: true},
-	{Name: "value", Type: bigquery.IntegerFieldType, Required: true},
-}
-
 // our test data has cardinality 5 for names, 3 for values
-var testSimpleData = []*testdata.SimpleMessage{
-	{Name: "one", Value: 1},
-	{Name: "two", Value: 2},
-	{Name: "three", Value: 3},
-	{Name: "four", Value: 1},
-	{Name: "five", Value: 2},
+var testSimpleData = []*testdata.SimpleMessageProto2{
+	{Name: proto.String("one"), Value: proto.Int64(1)},
```

Comment:
Why do we need these `proto.String` and `proto.Int64` wrapper types in proto2?

Reply:
The structs take pointers to strings rather than strings directly, the proto.String, proto.Int64 etc funcs are just wrappers to take a value and return a pointer to the value.